front/components/items/Appointment.tsx

```diff
@@ -6,35 +6,39 @@ import { DayPicker } from "react-day-picker";
 import { Grid } from "react-loader-spinner";
 
 const Appointment = () => {
-    const therapist = "Bożena"
+    const TEMPtherapist = "Bożena";
+    const filters: filters = {};
     let [chosenDate, setChosenDate] = useState<Date>();
-    let filters: filters = {};
+    let rightTime: Date;
+    let queryChosenDate: string | undefined;
+    useEffect(() => {
+        if (chosenDate) {
+            rightTime = new Date(chosenDate?.getTime() - (-120*60*1000));    
+            // changing timezone by reducing by 2h (because of polish timezone (GMT+2))
```

Comment:
It's a good way to write comment above the code, not under

Reply:
for the next time you can resolve conversation when you are sure that the problem is fixed